R/estimate_infections.R

```diff
@@ -245,6 +251,7 @@ estimate_infections <- function(reported_cases,
                                 CrIs = c(0.2, 0.5, 0.9),
                                 filter_leading_zeros = TRUE,
                                 zero_threshold = Inf,
+                                weigh_prior_delays = TRUE,
```

Comment:
Is this clear enough? As an alternative could cal it `weight_delay_prior_by_time` or similar? I think this doesn't matter hugely so happy as is.

Reply:
As a compromise I renamed it to `weigh_delay_priors` which I think is better.